aspnet/publishing/iis.rst

```diff
@@ -4,23 +4,37 @@ Publishing to IIS
 By `Rick Anderson`_
 
 In this article:
+	- `Configure the IIS server`_
 	- `Publish from Visual Studio`_
-	- `Xcopy to IIS Server`_
+	- `Xcopy to IIS server`_
+	- `Supported operating systems`_
+	- `Common errors`_
 	- `Additional Resources`_
-	
-Publish from Visual Studio  
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-1. Create an ASP.NET 5 app. In this sample, I'll create a MVC 6 app using the **Web Site** template under **ASP.NET 5 Preview Templates**. If you're not using the ``web`` and ``gen`` commands in your development and production work flow, you can remove them from the *project.json* file.
 
-.. code-block:: javascript
+Configure the IIS server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Install the HTTP Platform Handler:
+
+	- `64 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkID=690721>`_
+	- `32 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkId=690722>`_
+
+- Enable the **Web Server (IIS)** role and **Application Development** role services. Skip this step if you are on Windows 7 through Windows 10 (Client OSes), it's required only for servers.
```

Comment:
Cool ... but people will still have to use `Configuration Editor` to unlock the `<handlers>` section of `web.config`, correct? Maybe the installer for `HttpPlatformHandler` in the future can do that for everyone automatically.